examples/gallery/gallery.slint

```diff
@@ -1,170 +1,46 @@
 // Copyright © SixtyFPS GmbH <info@slint-ui.com>
 // SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
 
-import { ScrollView, Button, CheckBox, SpinBox, Slider, GroupBox, LineEdit, StandardListView,
-    ComboBox, HorizontalBox, VerticalBox, GridBox, TabWidget, TextEdit, AboutSlint } from "std-widgets.slint";
+import { CheckBox, StandardListView, StyleMetrics } from "std-widgets.slint";
+import { AboutPage, ControlsPage, ListViewPage, TextEditPage } from "ui/pages/pages.slint";
+import { GallerySettings } from "ui/gallery_settings.slint";
+import { Navigator } from "ui/navigator.slint";
+import { SideBar } from "ui/side_bar.slint";
 
 App := Window {
-    preferred-width: 500px;
-    preferred-height: 600px;
+  min-width: 760px;
+    min-height: 680px;
```

Comment:
ideally, the minimum size should be computed based on the minimum size of the contents by the layout.

Reply:
Yes I have to check a better way there.

Reply:
Should be fixed. Can you check it on Linux?